### PR TITLE
1835 interface to choose episode type

### PR DESCRIPTION
**Pull request by DaveBlueFrontier**

### CODE REVIEW

File: app/views/episodes/new.haml [hunk 1]
```diff
@@ -3,19 +3,16 @@
     %h2
       = link_to patients_path, class: 'side-link', title: 'Back' do
         = image_tag "arrow-left-white.png"
-      = form_title(@episode, 'Episode (TB)')
+      = form_title(@episode, 'Episode')
```
Comment (tripledub) [inline app/views/episodes/new.haml]: `I18n` ???

Comment (DaveBlueFrontier) [inline app/views/episodes/new.haml]: That doesn't show in the page as far as I can see.  I can't even see it in the page source!

Comment (tripledub) [inline app/views/episodes/new.haml]: Should either fix it or remove it then

Comment (DaveBlueFrontier) [inline app/views/episodes/new.haml]: Found out where it is used - the form_title function appends text to it before display.  Therefore i18n'd it.

**Pull request closed without merging**